src/components/homePage/Speakers.vue

```diff
@@ -1,19 +1,136 @@
 <template>
 <div id="speakers">
-	<h2>CUTC 2020</h2>
-	<!-- TAB LAYOUT -->
+	<div class="tabs">
+		<p class="tab" :class="[currentPage == 0 ? 'underlined' : '']" @click="currentPage = 0; toggleStyle()"> Speakers </p>
+		<p class="tab" :class="[currentPage == 1 ? 'underlined' : '']" @click="currentPage = 1; toggleStyle()"> Events </p>
+	</div>
+  
+	<vueper-slides v-if="currentPage == 0">
+		<template v-slot:arrow-right>
+			<img src="../../assets/Arrow.svg"/>
+		</template>
+		<vueper-slide 
+			v-for="(slide, i) in information['slides']" 
+			:key="i" >
+			<template v-slot:content>
+                  <img :src="getImage(slide.image)" id="person" :alt="slide.alt"/>
+                {{ slide.content }}
+        <div class="vueperslide__content-wrapper">
+        </div>
+      </template>
+		</vueper-slide>
+	</vueper-slides>
+	<vueper-slides v-else>
+		<template v-slot:arrow-right>
+			<img src="../../assets/Arrow.svg"/>
+		</template>
+		<vueper-slide 
+			v-for="(slide, i) in information['slides']" 
+			:key="i" >
+			<template v-slot:content>
+                  <img :src="getImage(slide.image)" id="person" :alt="slide.alt"/>
+                {{ slide.content }}
+        <div class="vueperslide__content-wrapper">
+        </div>
+      </template>
+		</vueper-slide>
+	</vueper-slides>
 </div>
 </template>
 
 <script>
+import { VueperSlides, VueperSlide } from 'vueperslides'
+import 'vueperslides/dist/vueperslides.css'
+import SpeakerInfo from "../../content/speakers.json";
+
 export default {
+	components: { VueperSlides, VueperSlide },
 	name: 'Speakers',
+	data: () => ({
+		information: SpeakerInfo,
+		currentPage: 0,
+}), 
+	methods: {
+		getImage(pic) {
+			try {
+				return require("../../assets/" + pic);
+			} catch(e) {
+				throw Error(`pic does not exist`);
```

Comment:
As mentioned in Sanaa's PR, while this statement is good for debugging, it shouldn't happen on the live website. Commenting it out works